src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts

```diff
@@ -51,6 +54,9 @@ export class ReportExtensionIssueAction extends Action {
 		unresponsiveProfile?: IExtensionHostProfile
 	}): Promise<string> {
 		let baseUrl = extension.marketplaceInfo && extension.marketplaceInfo.type === ExtensionType.User && extension.description.repository ? extension.description.repository.url : undefined;
+		if (extension.description.isBuiltin) {
```

Comment:
Also check if baseUrl is not set. This is needed for clairty
`if (!baseUrl &&`

Reply:
done